Xamarin.Forms.Core/Shell/SearchHandler.cs

```diff
@@ -504,6 +508,16 @@ static void OnCommandParameterChanged(BindableObject bindable, object oldValue,
 			((SearchHandler)bindable).OnCommandParameterChanged();
 		}
 
+		void ITextElement.OnCharacterSpacingPropertyChanged(double oldValue, double newValue)
+		{
+			
+		}
+
+		public void OnTextColorPropertyChanged(Color oldValue, Color newValue)
```

Comment:
why did you turned this implementation to implicit ?
```suggestion
		void ITextElement.OnTextColorPropertyChanged(Color oldValue, Color newValue)
```

Reply:
done